src/main/java/org/sonarlint/intellij/server/RequestProcessor.kt

```diff
@@ -84,6 +91,26 @@ class RequestProcessor(
         }
         return Success()
     }
+
+    private fun processUserToken(request: Request): Response {
+        val token = try {
+            parseToken(request)
```

Comment:
Is there any validation we should add here? Like a maximum size, to avoid vandalism?

Reply:
As discussed, I did not add any limit, but I restrict the processing of the token to only when the user initiated a token creation request. This limits the window